app/src/main/res/layout/app_bar_main.xml

```diff
@@ -118,21 +118,46 @@
                             android:visibility="invisible" />
                     </RelativeLayout>
 
-                    <RelativeLayout
-                        android:id="@+id/search_button"
-                        android:clickable="true"
-                        android:focusable="true"
-                        android:layout_width="40dp"
+                    <LinearLayout android:id="@+id/search_component"
+                        android:layout_width="wrap_content"
                         android:layout_height="match_parent"
-                        android:background="?attr/selectableItemBackground">
-                        <ImageView
-                            android:layout_width="24dp"
-                            android:layout_height="24dp"
-                            android:visibility="visible"
-                            android:layout_centerInParent="true"
-                            android:src="@drawable/ic_search"
-                            app:tint="@color/actionBarForeground" />
-                    </RelativeLayout>
+                        android:orientation="horizontal">
+                        <EditText android:id="@+id/search_query_text"
```

Comment:
I'd like to change this to an OutlinedTextBox (TextInputLayout style). We'll be going with this for all text input controls to maintain UI consistency.

It also needs to a be a single line input. The default action should start the search query (for when the user hits the keyboard action button).